lib/config.go

```diff
@@ -335,9 +354,8 @@ func DefaultConfig() *Config {
 			FileLockingPath: lockPath,
 		},
 		RuntimeConfig: RuntimeConfig{
-			Runtime:                  "/usr/bin/runc",
-			RuntimeUntrustedWorkload: "",
-			DefaultWorkloadTrust:     "trusted",
+			Runtime:  "/usr/bin/runc",
```

Comment:
this should go away and we need just:
```
-                       Runtime:  "/usr/bin/runc",
-                       Runtimes: map[string]oci.RuntimeHandler{},
-
+                       Runtimes: map[string]oci.RuntimeHandler{
+                               "runc": {
+                                       RuntimePath: "/usr/bin/runc"
+                               },
+                       },
```

Reply:
basically, Runtime should only be used as fallback for compatibily, otherwise, always prefer RuntimeHandler